examples/lighting-app/efr32/BUILD.gn

```diff
@@ -44,6 +44,12 @@ declare_args() {
 
   # Monitor & log memory usage at runtime.
   enable_heap_monitoring = false
+  # Wifi related stuff - they are overriden by gn -args="use_wf200=true"
```

Comment:
please add build_examples.py support for these boards as well, likely as build variants.

Reply:
I will take care of this in a subsequent PR once this gets merge. I am familiar with build_examples